app/facades/quotes_facade.rb

```diff
@@ -0,0 +1,10 @@
+class QuotesFacade
+  def self.keyword_quotes_search(keyword)
+    keyword_search = QuotesService.keyword_quotes(keyword)
+    {
+      keyword: keyword,
+      total_results: keyword_search[:totalCount],
+      quotes: keyword_search[:results].map { |quote| Quote.new(quote)}
+    }
+  end
```

Comment:
You could also put this into some sort of super object to keep track of Search Results